.github/workflow-templates/chainguard-go-test.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 The Chainguard Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is automagically synced here from github.com/Chainguard-sandbox/.github
+# repo by knobots: https://github.com/Chainguard-sandbox/knobots and will be overwritten.
```

Comment:
This is definitely stale.  I'd probably kill most of these workflow templates (we definitely don't want to have to maintain forks of our own actions).  We can potentially rework the ones we care about to use the new reusable workflows via `workflow_call`.

I also think copying all of this under our own copyright header without attribution is bad form.  If we're going to fork these, we should provide attribution to the Knative authors, but frankly IDK that I've ever seen the whole `workflow-templates` directory provide any value (we mostly just used it as a central place to auto-sync from in Knative).